From 4773ee3452fb476873f874bf4ed3db07f4b3297f Mon Sep 17 00:00:00 2001
From: zhangyichix <yichix.zhang@intel.com>
Date: Tue, 19 Sep 2023 09:16:02 +0800
Subject: [PATCH] libstagefright/foundation/include path changed

libstagefright/foundation has been moved to module
Updating the same.

Tracked-On: OAM-112296
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Signed-off-by: zhangyichix <yichix.zhang@intel.com>
---
 Android.bp          | 4 ++--
 c2_utils/Android.bp | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Android.bp b/Android.bp
index e255cba..e5f3acd 100644
--- a/Android.bp
+++ b/Android.bp
@@ -63,8 +63,8 @@ mfx_cc_defaults {
 
     include_dirs: [
         "hardware/intel/external/minigbm-intel/cros_gralloc", // remove then minigbm gets Android.bp
-        "frameworks/av/media/libstagefright/foundation/include",
-        "frameworks/av/media/bufferpool/2.0/include",
+        "frameworks/av/media/module/foundation/include",
+        "frameworks/av/media/module/bufferpool/2.0/include",
         "frameworks/av/media/codec2/sfplugin/utils",
         "frameworks/av/media/codec2/vndk/include",
         "frameworks/native/include",
diff --git a/c2_utils/Android.bp b/c2_utils/Android.bp
index 25438ad..c24015e 100644
--- a/c2_utils/Android.bp
+++ b/c2_utils/Android.bp
@@ -62,7 +62,7 @@ cc_library_static {
         "frameworks/av/media/codec2/components/base/include",
         "frameworks/av/media/codec2/core/include",
         "frameworks/av/media/libstagefright/include",
-        "frameworks/av/media/libstagefright/foundation/include",
+        "frameworks/av/media/module/foundation/include",
         "frameworks/native/libs/gralloc/types/include",
         "system/core/include/utils",
         "external/libdrm/include/drm",
-- 
2.40.0

